frontend/src/utils/routes.ts

```diff
@@ -0,0 +1,6 @@
+export const Home = '/';
+
+export const Path = {
+  form: Home + 'form',
+  chart: Home + 'chart',
```

Comment:
Don't do a string addition, use the backticks. And does your Home location will change ? I don't think no, so you don't have to make some compute here.

By conventions, we prefer to put all our routes in a single object.